Source/AliveLibAE/Input.cpp

```diff
@@ -1264,6 +1264,24 @@ ALIVE_VAR(1, 0x5c9f74, DWORD, sPrevious_down_keyboard_keys_5C9F74, 0);
 ALIVE_VAR(1, 0x5c9f78, DWORD, dword_5C9F78, 0);
 ALIVE_VAR(1, 0x5c9794, int, sKeyboardBindings_5C9794, 0);
 
+bool IsChantingAnyShoulderButton(int shoulderButtonsPressedCount)
+{
+    if (shoulderButtonsPressedCount > 1)
```

Comment:
can just be return shoulderButtonsPressedCount > 1; ? this function can be static too